### ISSUE TITLE
Hard coded Speed Estimation 

### ISSUE THREAD

**Issue by MrFahad** [roboflow/supervision#734]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar bug report.

### Bug

Thank you so much for such great work. I have test all of the 3 scenarios, i.e. NAS, Inference, ultralytics.

The code is working perfectly.

But I have one question, how to make source and target dynamic so that they can handle any kind of resolution video. ??

I have seen the code is working great but with only vehicles.mp4 , because you have set the values according to that video 3480x2160.

Thanks,

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
Hi, @MrFahad 👋🏻 ! I think the only hardcoded part in the script is the `SOURCE` and `TARGET` definitions. Is that what you are referring to? 

If so, those parameters must be adjusted separately for every camera view. 

Could you share more about what you would expect from a better example script?

**Comment by MrFahad** [comment]
Yes that's what I was saying, how to make them dynamic?? Can you please give me an example code that how to adjust them for every camera view??

**Comment by SkalskiP** [comment]
> Yes, that's what I was saying; how do you make them dynamic?? Can you please give me an example code to adjust them for every camera view??

@MrFahad, the problem is you can't make those dynamically. You must provide this configuration for each camera to make an accurate speed estimation. Those measurements must most likely be made manually.

**Comment by MrFahad** [comment]
So, what if we have 10 cameras then we need to configure each camera manually. Am I right??

And secondly, for accurate speed Estimation will we take average of all the cameras view.??

Let's say we have 2 cameras how you will write it manually???

**Comment by SkalskiP** [comment]
> So, what if we have 10 cameras then we need to configure each camera manually. Am I right??

Yes you need to make the configuration manually. You only need to do it once for each camera. Then, just store it in a JSON file. 

> And secondly, for accurate speed Estimation will we take average of all the cameras view.??

That's an extra step. You don't need to do it.

**Comment by SkalskiP** [comment]
Btw, I'm converting this issue into a discussion and putting it into the Q&A section.
